presto-docs/src/main/sphinx/functions/binary.rst

```diff
@@ -53,6 +53,11 @@ Binary Functions
     negative starting position is interpreted as being relative to the end of
     the string.
 
+.. function:: reverse(binary) -> varbinary
```

Comment:
also add to the list of functions

Reply:
@mosabua I'm not too familiar with the docs. `functions.rst` already includes this `binary.rst` file. Where is the list of functions that I should add to?

Reply:
@mosabua is referring to `presto-docs/src/main/sphinx/functions/list.rst`

But we can't add it there, since the link for `reverse` already goes to the string version.

Reply:
hm ... need to find a better name for the function then I guess .. or another way to deal with this name overlap with some sort of disambiguation or so .. any suggestions @electrum ?

Reply:
Do you need me to do anything as part of this PR to address this?

I see there are a few other functions with string and binary versions that have this issue e.g. `substr`.